src/v2/Apps/Home/homeRoutes.tsx

```diff
@@ -20,8 +20,8 @@ export const homeRoutes: AppRouteConfig[] = [
         homePage {
           ...HomeApp_homePage
         }
-        orderedSet(id: "529939e2275b245e290004a0") {
-          ...HomeApp_orderedSet
+        featuredEventsOrderedSet: orderedSet(id: "529939e2275b245e290004a0") {
```

Comment:
Renamed this to be a bit more specific since we're fetching with the main query and now have multiple ordered sets coming through